notifications_utils/template.py

```diff
@@ -421,8 +428,16 @@ def __str__(self):
 
 
 class SubjectMixin:
```

Comment:
[comment] This mixing is starting to make my head go a bit 🤯 it doesn't feel so independent or self-contained any more.

Reply:
yeah 😅